### PR TITLE
blog: Knowledge Sharing Platform

### PR DESCRIPTION
**Pull request by battuAshita**
Please review my blog and accept this PR

### CODE REVIEW

File: content/blog/knowledge_sharing_platform.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: Foss Overflow'21: My Experience
```
Comment (supercoww) [inline content/blog/knowledge_sharing_platform.md:2]: Try wrapping the title in quotes. `"Foss Overflow'21: My Experience"`

Comment (battuAshita) [reply content/blog/knowledge_sharing_platform.md:2]: Okay

**Comment by ashutosh321607** [comment]
@battuAshita you can add some of the UI designs that you had created and the URL of the current project. Otherwise looks good to me, you did an awesome job.
All the best for your future endeavors.

@ExpressHermes any review from your side?

**Comment by battuAshita** [comment]
> @battuAshita you can add some of the UI designs that you had created and the URL of the current project. Otherwise looks good to me, you did an awesome job. All the best for your future endeavors.
> 
> @ExpressHermes any review from your side?

Done.

**Review by ashutosh321607 — APPROVED** [review]

**Review by ExpressHermes — APPROVED** [review]
LGTM!

**Pull request merged**